services/auth/source/ldap/source.go

```diff
@@ -55,6 +55,9 @@ type Source struct {
 	GroupMemberUID        string // Group Attribute containing array of UserUID
 	UserUID               string // User Attribute listed in Group
 	SkipLocalTwoFA        bool   `json:",omitempty"` // Skip Local 2fa for users authenticated with this source
+	TeamGroupMap          string // Map LDAP groups to teams
```

Comment:
One slight fly in the ointment here is that the login_source will be read and loaded from the source quite a lot - it's infact in the DB as a JSON where it gets unmarshaled on load.